scripts/diff-features.js

```diff
@@ -4,6 +4,16 @@ const fs = require('fs');
 const yargs = require('yargs');
 
 function main({ ref1, ref2, format, github }) {
+  const results = diff({ ref1, ref2, github });
+
+  if (format === 'markdown') {
+    printMarkdown(results);
+  } else {
+    console.log(JSON.stringify(results, undefined, 2));
+  }
+}
+
+function diff({ ref1, ref2, github }) {
```

Comment:
I've extracted the actual `diff` portion here to an exported function, so it can be used in the `release-notes` script.